services/QuillLMS/client/app/bundles/Staff/components/evidence/datasetEditor/components/examples/exampleRow.tsx

```diff
@@ -111,7 +109,7 @@ const ExampleRowComponent: React.FC<ExampleRowProps> = ({
     onUpdate(example.id, example.usage, 'usage', e.target.value);
   };
 
-  const rowClassName = `example-row ${clusterIndex % 2 === 0 ? 'even-cluster' : 'odd-cluster'}`;
```

Comment:
When I interviewed at Quill, one of the questions I remember them asking me was "how do you feel about zebra stripes?" 😆